addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1086,7 +1095,7 @@ const Wysiwyg = Widget.extend({
      * Stop the field changes mutation observers.
      */
     _pauseOdooFieldObservers: function () {
-        for (let observerData of this.odooFieldObservers) {
+        for (let observerData of this.__externalOdooFieldObservers) {
```

Comment:
Maybe add a fallback on the old `this.odooFieldObservers` in case there are other calls to this method that did not replace the value ?